test/distributed/_tensor/test_utils.py

```diff
@@ -27,36 +24,6 @@ class UtilTest(DTensorTestBase):
     def world_size(self):
         return 8
 
-    @with_comms
-    def test_compute_local_shape_2d_uneven(self):
```

Comment:
It seems here there are some interesting uneven cases. Should we keep them? E.g. we can just rename this function and switch function calls.

Reply:
merged those tests into `test_compute_local_shape_and_global_offset_2D`